README.md

```diff
@@ -1 +1,62 @@
-# To Do List in JavaScript
+## To Do List in JavaScript
+
+This is the second project in the Microverse's JavaScript module.
+
+
+### Description
+"To DO List" is a simple HTML. CSS & Javascript application that enables a user to add, remove and re-order to-do lists.
+
+### Live Link
+
+[Live Link](https://kimothokamau.github.io/todo_list/)
+
+
+### Learning objectives
+- Use webpack to bundle JavaScript.
+- Learn how to use proper ES6 syntax.
+- Use ES6 modules to write modular JavaScript.
+
+
+## Built With
+
+- JavaScript
+- HTML & CSS
+
+### Prerequisites
+
+* Internet connection
+* You need to have a browser
+* Install node and npm
+
+## Getting started
+
+- Run this command in your OS terminal: `https://github.com/kimothokamau/todo_list` to get a local copy of the project.
+- Navigate to the project directory
+- Run `npm install`
+- Run `npm run build`
+- Navigate to the dist folder (`cd dist`) and open the index.html file in your favorite browser. Preferably chrome
+- Confirm that you can see a list of todo lists
+
+
+## Authors
+
+- GitHub: [kimothokamau](https://github.com/kimothokamau)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/disc3110/Miro-reddit/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- To developers, Microverse, ruby official guidelines and The odin Project
+
+## 📝 License
+
+This project is [MIT](https://es.wikipedia.org/wiki/Licencia_MIT) licensed.
```

Comment:
- [ ] Kindly follow this [guideline](https://github.com/microverseinc/readme-template) for how to generate a MIT license link. 😊